### ISSUE TITLE
Некорректно заполняется поле Документы об отгрузке 5(а) у СФ и УПД, оформленных после 01.07.2021! 

### ISSUE THREAD

**Issue by vlad3190** [diadoc/diadoc1cpro-docs#57]
Коллеги, добрый день! В обработке Диадок Про, версии 5.49.11, неккоректно заполняется поле "Документы об отгрузке" 5(а). Вот описание порядка заполнения: https://www.glavbukh.ru/art/101674-kak-zapolnit-stroku-5a-scheta-faktury-na-realizatsiyu-s-1-iyulya-2021-goda

Выдержка:![Screenshot_20210723-074543_Samsung Internet.jpg](https://user-images.githubusercontent.com/34089439/126739296-d84c8a75-50f6-4dc8-b42a-45b5e397ab81.jpg)

Необходимо срочно исправить!

**Comment by kulakovskaya** [comment]
Добрый день! Правильно понимаем, что под некорректным заполнением подразумеваете отсутствие данных по порядковым номерам товара "№ п/п" в документах об отгрузке?

**Comment by vlad3190** [comment]
Да, именно так! Я понимаю, что поле отсутсвует в схеме 820 приказа, но бухгалтера смотрят на 534 приказ, и хотят видеть эти номера в визуализации печатной формы, как в 1С. Вы не задавали вопрос в ведомства?

**Comment by volodkindv** [comment]
Все верно. В схеме 820 формата поле отсутствует, и здесь явная нестыковка с требованиями к печатной форме документа. Вопрос в ведомство задавали, ждем ответа.

**Comment by acharushkin** [comment]
Здравствуйте!

Мы архивируем проект `diadoc1cpro-docs`.

Актуальная документация по модулю Диадока для 1С находится здесь:

- [Для разработчиков](https://developer.kontur.ru/docs/diadoc_um/index.html)
- [Для пользователей](https://support.kontur.ru/diadoc-1s8x)

Наши контакты:

- [Клиентская поддержка Контур.Диадок](https://kontur.ru/support#diadoc) - для консультации по ошибкам в работе модуля
- [Заявка на интеграцию](https://www.diadoc.ru/order1c#order) - для подключения интеграции и связанных вопросов. Например, если нужен сертификат, узнать тарифы и стоимость подключения

С уважением, команда разработки Диадока
